src/dart/parse.rs

```diff
@@ -154,14 +165,19 @@ impl<'a> Parser<'a> {
         Ok(())
     }
 
+    /// Returns true and consumes the current token
+    /// if it matches the given punctuation pair.
     pub fn eat_punctuation2(&mut self, c1: char, c2: char) -> bool {
         self.try(|p| p.expect_punctuation2(c1, c2)).is_some()
     }
 
+    /// Returns true if the current token matches the given punctuation pair.
     fn is_punctuation2(&self, c1: char, c2: char) -> bool {
         self.probe(|p| p.eat_punctuation2(c1, c2))
     }
 
+    /// Consumes the current tokens if it matches the
```

Comment:
"current tokens" is not accurate. "Consumes a number of tokens if they match ..."